engine/src/org/pentaho/di/trans/steps/rest/Rest.java

```diff
@@ -335,10 +317,6 @@ public boolean verify( String hostname, SSLSession session ) {
     }
   }
 
-
-  protected MultivaluedMap<String, String> searchForHeaders( ClientResponse response ) {
```

Comment:
The [RestIT class](https://github.com/pentaho/pentaho-kettle/blob/master/engine/test-src/org/pentaho/di/trans/steps/rest/RestIT.java#L110) failed to compile because a method is said to Override this now-removed method in the Rest class.

Is there a need to remove this and other areas of this class to handle self-signed SSL certificates?